demisto_sdk/commands/common/tests/pack_metadata_validator_test.py

```diff
@@ -96,3 +96,19 @@ def test_name_does_not_contain_excluded_word(self):
     def read_file(file_):
         with io.open(file_, mode="r", encoding="utf-8") as data:
             return data.read()
+
+    def test_metadata_not_dict(self, mocker):
```

Comment:
why is this relevant to the PR?

Reply:
Added for coverage, saw a line that wasn't covered and added it